contrib/automation_tests/test_cases/symbols_tab.py

```diff
@@ -133,7 +76,11 @@ def __init__(self, **kwargs):
 
     def _execute(self, expected_duration_difference_ratio: float = None):
         """
-        :param expected_duration_difference: @see DurationDiffMixin._check_and_update_duration
+        :param expected_difference_ratio: Expected difference as ratio relative to the previously stored value.
+            If < 1.0, the current run must take *at most* expected_difference_ratio * previous_time.
+            If >= 1.0, the current run must take *at least* expected_difference_ratio * previous_time.
+            If None, this will only update the stored value.
+            Violating those conditions will result in test failure.
```

Comment:
Nit: This comment explains in great detail what's happening except on the first run. What happens when there is no previously stored value and the expected_duration_difference_ratio is set?

Reply:
Good point, I've added this. The new value is stored, but no comparison is made and the test can't fail.